content/package.json

```diff
@@ -12,7 +12,10 @@
     "chokidar": "3.3.1",
     "dotenv": "8.2.0",
     "fast-csv": "4.1.3",
+    "got": "10.7.0",
+    "html-differ": "^1.4.0",
```

Comment:
Curious that this got added with a `^` but the `got` one above didn't. 
How did you add these two?

Reply:
Hm, I must have added `got` manually (shame on me!). I'll change that. Thanks!

Reply:
We actually have a think in CI that protects from that. So you would have gotten caught sooner or later :)